src/java/src/main/java/com/dnanexus/DXDataObject.java

```diff
@@ -42,7 +42,10 @@
 import com.google.common.collect.Maps;
 
 /**
- * Base class for all data object classes in the DNAnexus Platform.
+ * Base class for all data object classes in the DNAnexus
+ * Platform. Instances of this class are JSON-serialized as a DNAnexus
```

Comment:
(Here and elsewhere...) Per the [Javadoc conventions](http://www.oracle.com/technetwork/articles/java/index-137868.html), the first sentence of the docstring should stand alone (since the class summary or method list will just show this one sentence) and subsequent commentary should be set off with `<p>...</p>`.